apps/main/templates/event/event_confirm_delete.html

```diff
@@ -0,0 +1,8 @@
+{% extends '_base.html' %}
+{% block title %}Vibely - Delete event{% endblock title %}
+{% block content %}
+<form method="post">{% csrf_token %}
+    <p>Are you sure you want to delete the event "{{ object.name }}"?</p>
+    <input type="submit" value="Confirm">
+</form>
+{% endblock content %}
```

Comment:
The conventions was templates/event/method, so it should be named templates/event/event.html